vs-code/vscode.md

```diff
@@ -12,7 +12,11 @@ About 1 hour
 
 ### Motivation
 
-Most software is written using text editors. Visual Studio Code (VS Code) is a very popular open-source text editor developed and maintained by Microsoft. Having a good understanding of how to use VS Code will enable you to code and use other text editors with ease.
+Most software is written using text editors. Visual Studio Code (VS Code) is a very popular open-source text editor developed and maintained by Microsoft. Having a good understanding of how to use VS Code will enable you to code and use other text editors with ease. Most of the developers prefer using VS Code for writing codes because of its flexiblity and user interface. It provides almost all the facilities that makes the taks easier.
```

Comment:
Many developers prefer using VSCode for developmental because of its...
It provides lots of tools to make tasks easier.